ext/bcmath/bcmath.c

```diff
@@ -1280,8 +1283,15 @@ static int bcmath_number_compare(zval *op1, zval *op2)
 
 	return (int) ret;
 
-fallback:
-	return zend_std_compare_objects(op1, op2);
+	failure:
```

Comment:
If you declare `int ret = ZEND_UNCOMPARABLE;` at the start of the function, then you can put the failure label at line 1276, which would reduce code duplication.

Reply:
Done, sorry for the delay